package.json

```diff
@@ -14,14 +14,16 @@
     "eslint-config-airbnb-base": "^7.0.0",
     "eslint-config-angular": "^0.5.0",
     "eslint-config-ember": "^0.3.0",
-    "eslint-plugin-ember-suave": "^1.0.0",
     "eslint-config-google": "^0.6.0",
     "eslint-config-hapi": "^10.0.0",
     "eslint-config-standard": "^6.0.0",
     "eslint-config-standard-jsx": "^3.0.0",
     "eslint-config-standard-react": "^4.0.0",
+    "eslint-import-resolver-node": "https://registry.npmjs.org/eslint-import-resolver-node/-/eslint-import-resolver-node-0.2.3.tgz",
```

Comment:
This package appears to be actually released, why do we need to install via a tgz like this?